app/helpers/application_helper.rb

```diff
@@ -311,6 +311,15 @@ def crawlable_meta_data(opts = nil)
         result << tag(:meta, { name: "twitter:#{property}", content: content }, nil, true)
       end
     end
+    Array
+      .wrap(opts[:breadcrumbs])
+      .each do |breadcrumb|
+        result << tag(:meta, property: "og:article:section", content: breadcrumb[:name])
```

Comment:
Should we check that `breadcrumb[:name]` is `.present?` before adding a meta tag or are we sure there will always be a value there?

Reply:
The `breadcrumbs` option is only used with the new code and expects to come from the breadcrumbs we already have in the topic view. If `breadcrumbs` is provided, it should contain both `name` and `color` keys. Otherwise, we’re using `Array.wrap(opts[:breadcrumbs])` so this way if the option isn’t provided the result is an empty array which in turn results in doing nothing.